.github/workflows/ql-for-ql-build.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - uses: actions/checkout@v3
       - name: Find codeql
         id: find-codeql
-        uses: github/codeql-action/init@aa93aea877e5fb8841bcb1193f672abf6e9f2980
+        uses: github/codeql-action/init@71a8b35ff4c80fcfcd05bc1cd932fe3c08f943ca
```

Comment:
Could we point this at a tag to make similar updates simpler.

I.e. create a tag for 71a8b35ff4c80fcfcd05bc1cd932fe3c08f943ca on gitub/codeql-action titled `v2-with-ql-language`, and use that instead of the explicit sha.